{
  "Title": "Security update for bind",
  "Tracking": {
    "ID": "SUSE-SU-2022:2713-1",
    "Status": "Final",
    "Version": "1",
    "InitialReleaseDate": "2022-08-09T10:38:14Z",
    "CurrentReleaseDate": "2022-08-09T10:38:14Z",
    "RevisionHistory": [
      {
        "Number": "1",
        "Date": "2022-08-09T10:38:14Z",
        "Description": "current"
      }
    ]
  },
  "Notes": [
    {
      "Text": "Security update for bind",
      "Title": "Topic",
      "Type": "Summary"
    },
    {
      "Text": "This update for bind fixes the following issues:\n\n- CVE-2021-25219: Fixed flaw that allowed abusing lame cache to severely degrade resolver performance (bsc#1192146).\n- CVE-2021-25220: Fixed potentially incorrect answers by cached forwarders (bsc#1197135).\n- CVE-2022-0396: Fixed a incorrect handling of TCP connection slots time frame  leading to deny of service (bsc#1197136).\n\nThe following non-security bugs were fixed:\n\n- Update to release 9.16.31 (jsc#SLE-24600). \n- Logrotation broken since dropping chroot (bsc#1200685).\n- A non-existent initialization script (eg a leftorver\n  'createNamedConfInclude' in /etc/sysconfig/named) may cause named\n  not to start. A warning message is printed in named.prep and\n  the fact is ignored.\n  Also, the return value of a failed script was not handled properly\n  causing a failed script to not prevent named to start. This\n  is now fixed properly.\n  [bsc#1199044, vendor-files.tar.bz2]\n",
      "Title": "Details",
      "Type": "General"
    },
    {
      "Text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
      "Title": "Terms of Use",
      "Type": "Legal Disclaimer"
    }
  ],
  "ProductTree": {
    "Relationships": [
      {
        "ProductReference": "bind-utils-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Basesystem 15 SP4",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python3-bind-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Basesystem 15 SP4",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "bind-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Server Applications 15 SP4",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "bind-doc-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "SUSE Linux Enterprise Module for Server Applications 15 SP4",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "bind-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "openSUSE Leap 15.4",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "bind-doc-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "openSUSE Leap 15.4",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "bind-utils-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "openSUSE Leap 15.4",
        "RelationType": "Default Component Of"
      },
      {
        "ProductReference": "python3-bind-9.16.31-150400.5.6.1",
        "RelatesToProductReference": "openSUSE Leap 15.4",
        "RelationType": "Default Component Of"
      }
    ]
  },
  "References": [
    {
      "URL": "https://www.suse.com/support/update/announcement/2022/suse-su-20222713-1/",
      "Description": "Link for SUSE-SU-2022:2713-1"
    },
    {
      "URL": "https://lists.suse.com/pipermail/sle-security-updates/2022-August/011832.html",
      "Description": "E-Mail link for SUSE-SU-2022:2713-1"
    },
    {
      "URL": "https://www.suse.com/support/security/rating/",
      "Description": "SUSE Security Ratings"
    },
    {
      "URL": "https://bugzilla.suse.com/1192146",
      "Description": "SUSE Bug 1192146"
    },
    {
      "URL": "https://bugzilla.suse.com/1197135",
      "Description": "SUSE Bug 1197135"
    },
    {
      "URL": "https://bugzilla.suse.com/1197136",
      "Description": "SUSE Bug 1197136"
    },
    {
      "URL": "https://bugzilla.suse.com/1199044",
      "Description": "SUSE Bug 1199044"
    },
    {
      "URL": "https://bugzilla.suse.com/1200685",
      "Description": "SUSE Bug 1200685"
    },
    {
      "URL": "https://www.suse.com/security/cve/CVE-2021-25219/",
      "Description": "SUSE CVE CVE-2021-25219 page"
    },
    {
      "URL": "https://www.suse.com/security/cve/CVE-2021-25220/",
      "Description": "SUSE CVE CVE-2021-25220 page"
    },
    {
      "URL": "https://www.suse.com/security/cve/CVE-2022-0396/",
      "Description": "SUSE CVE CVE-2022-0396 page"
    }
  ],
  "Vulnerabilities": [
    {
      "CVE": "CVE-2021-25219",
      "Description": "In BIND 9.3.0 -\u003e 9.11.35, 9.12.0 -\u003e 9.16.21, and versions 9.9.3-S1 -\u003e 9.11.35-S1 and 9.16.8-S1 -\u003e 9.16.21-S1 of BIND Supported Preview Edition, as well as release versions 9.17.0 -\u003e 9.17.18 of the BIND 9.17 development branch, exploitation of broken authoritative servers using a flaw in response processing can cause degradation in BIND resolver performance. The way the lame cache is currently designed makes it possible for its internal data structures to grow almost infinitely, which may cause significant delays in client query processing.",
      "Threats": [
        {
          "Type": "Impact",
          "Severity": "important"
        }
      ],
      "References": [
        {
          "URL": "https://www.suse.com/security/cve/CVE-2021-25219.html",
          "Description": "CVE-2021-25219"
        },
        {
          "URL": "https://bugzilla.suse.com/1192146",
          "Description": "SUSE Bug 1192146"
        }
      ],
      "ProductStatuses": [
        {
          "Type": "Fixed",
          "ProductID": [
            "SUSE Linux Enterprise Module for Basesystem 15 SP4:bind-utils-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Basesystem 15 SP4:python3-bind-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Server Applications 15 SP4:bind-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Server Applications 15 SP4:bind-doc-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-doc-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-utils-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:python3-bind-9.16.31-150400.5.6.1"
          ]
        }
      ],
      "CVSSScoreSets": {}
    },
    {
      "CVE": "CVE-2021-25220",
      "Description": "BIND 9.11.0 -\u003e 9.11.36 9.12.0 -\u003e 9.16.26 9.17.0 -\u003e 9.18.0 BIND Supported Preview Editions: 9.11.4-S1 -\u003e 9.11.36-S1 9.16.8-S1 -\u003e 9.16.26-S1 Versions of BIND 9 earlier than those shown - back to 9.1.0, including Supported Preview Editions - are also believed to be affected but have not been tested as they are EOL. The cache could become poisoned with incorrect records leading to queries being made to the wrong servers, which might also result in false information being returned to clients.",
      "Threats": [
        {
          "Type": "Impact",
          "Severity": "important"
        }
      ],
      "References": [
        {
          "URL": "https://www.suse.com/security/cve/CVE-2021-25220.html",
          "Description": "CVE-2021-25220"
        },
        {
          "URL": "https://bugzilla.suse.com/1197135",
          "Description": "SUSE Bug 1197135"
        }
      ],
      "ProductStatuses": [
        {
          "Type": "Fixed",
          "ProductID": [
            "SUSE Linux Enterprise Module for Basesystem 15 SP4:bind-utils-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Basesystem 15 SP4:python3-bind-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Server Applications 15 SP4:bind-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Server Applications 15 SP4:bind-doc-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-doc-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-utils-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:python3-bind-9.16.31-150400.5.6.1"
          ]
        }
      ],
      "CVSSScoreSets": {}
    },
    {
      "CVE": "CVE-2022-0396",
      "Description": "BIND 9.16.11 -\u003e 9.16.26, 9.17.0 -\u003e 9.18.0 and versions 9.16.11-S1 -\u003e 9.16.26-S1 of the BIND Supported Preview Edition. Specifically crafted TCP streams can cause connections to BIND to remain in CLOSE_WAIT status for an indefinite period of time, even after the client has terminated the connection.",
      "Threats": [
        {
          "Type": "Impact",
          "Severity": "important"
        }
      ],
      "References": [
        {
          "URL": "https://www.suse.com/security/cve/CVE-2022-0396.html",
          "Description": "CVE-2022-0396"
        },
        {
          "URL": "https://bugzilla.suse.com/1197136",
          "Description": "SUSE Bug 1197136"
        }
      ],
      "ProductStatuses": [
        {
          "Type": "Fixed",
          "ProductID": [
            "SUSE Linux Enterprise Module for Basesystem 15 SP4:bind-utils-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Basesystem 15 SP4:python3-bind-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Server Applications 15 SP4:bind-9.16.31-150400.5.6.1",
            "SUSE Linux Enterprise Module for Server Applications 15 SP4:bind-doc-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-doc-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:bind-utils-9.16.31-150400.5.6.1",
            "openSUSE Leap 15.4:python3-bind-9.16.31-150400.5.6.1"
          ]
        }
      ],
      "CVSSScoreSets": {}
    }
  ]
}